tests/array_in_struct_test/schema/uuid.fbs

```diff
@@ -0,0 +1,4 @@
+
+struct Uuid {
```

Comment:
rather than adding a new schema, I believe we already have existing schemas being used in tests for arrays.

Reply:
I did not find any example where arrays are used in structs

Reply:
thank you!. I updated the examples @aardappel